src/index.ts

```diff
@@ -1,3 +1,5 @@
+export * from './arithmetic';
 export * from './dimension';
 export * from './exponent';
+export * from './geometric';
```

Comment:
@buge 

While testing the library on one of my personal project, I spotted a caveat.

In my project, I am using only `makeUnitFactory` in order to create my own units.
However, when I look at my generated bundle, I can see I am bundling `NativeAlgorithm` and `NativeGeometric`.

More generally, if someone want to use `withValueType` of any units, he will get in the bundle `NativeAlgorithm`.
Even if he does not care about.

By having the function `withValueType` and the native definition in the same file, I am not sure we can avoid that :(

Any opinion/suggestion ?

Reply:
I don't see a good way of achieving this without breaking backwards compatibility. I think what you want to do in these cases is use a good tree-shaking builder like Webpack to trim any code that isn't used in your case. I haven't actively tried to use Webpack or anything similar on projects that use this library, so there's certainly optimizations that could be done (like setting `"sideEffects": false`). I'd suggest we merge as-is and then can follow-up as needed.